README.md

```diff
@@ -5,7 +5,8 @@
 ### Building:
 
 ```bash
-git clone https://github.com/opencontainers/runc
+go get -d github.com/opencontainers/runc
+cd $GOPATH/src/github.com/opencontainers/runc/
```

Comment:
In my PR ( #23) I just put:
\#cd to `runc` dir
not ideal, but at least its accurate.  Thoughts?